src/renderer/services/notification/index.js

```diff
@@ -9,17 +9,17 @@ const INON_HASH = {
   info: 'icon-info'
 }
 
-const COLOR_HASH = {
-  primary: 'var(--themeColor)',
-  error: 'var(--deleteColor)',
-  warning: 'var(--deleteColor)',
-  info: '#999999'
-}
+// const COLOR_HASH = {
+//   primary: 'var(--themeColor)',
+//   error: 'var(--deleteColor)',
+//   warning: 'var(--deleteColor)',
+//   info: '#999999'
+// }
 
 const notification = {
   name: 'notify',
   noticeCache: {},
-  // it's a dirty implement of clear, because not remove all the event listeners. need refactor.
+  // TODO: it's a dirty implement of clear, because not remove all the event listeners. need refactor.
```

Comment:
my fault, I think we can remove this comment now, because we remove all the event listeners.